src/main/kotlin/eli/inevitable/Inevitable.kt

```diff
@@ -0,0 +1,156 @@
+package eli.inevitable
+
+import kotlin.text.RegexOption.DOT_MATCHES_ALL
+
+/**
+ * System agnostic character sequence for generating a line break.
+ */
+val LINE_BREAK = System.lineSeparator()
+/**
+ * Regex pattern for identifying and breaking up new lines.
+ */
+val LINE_BREAK_PATTERN = Regex("""(\r\n|\r|\n)""", DOT_MATCHES_ALL)
+
+/**
+ * Regex pattern for identifying and replacing tab characters.
+ */
+val TAB_PATTERN = Regex("""\t""", DOT_MATCHES_ALL)
+
+/**
+ * Regex pattern for identifying all non-printable characters (excluding space ' ').
+ */
+val NON_DISPLAY_PATTERN = Regex("""^[\S ]+$""", DOT_MATCHES_ALL)
+
+/**
+ * Extension function to determine the max width of a [collection of strings][Collection]
+ */
+fun Collection<String>.maxWidth(): Int = this.maxBy { it.length }?.length ?: 0
+
+/**
+ * Breaks apart a single [string of text][String] into multiple lines if there are any line
+ * breaks, converts all tab characters into spaces, and trims any extra whitespace off the end
+ * of each line.
+ *
+ * @return A [List] of sanitized [strings][String].
+ */
+fun String.sanitize(tabSpaces: Int = 2): List<String> = LINE_BREAK_PATTERN.split(this).map {
```

Comment:
do defaulted params get auto ktdoc'd ? or do we need to specify `@param`

Reply:
I need to kdoc them. I've been trying to add where I see them, but I've missed a few. Thanks, I'll double check.